app/src/userpages/components/Header/Tab/tab.pcss

```diff
@@ -1,11 +1,11 @@
 .root {
   font-family: var(--mono);
-  font-size: calc(0.75 * var(--um));
+  font-size: 0.75um;
```

Comment:
It's simply easier to use than the actual `calc`+`var`.

Reply:
cool